chsdi/templates/htmlpopup/kgs_anhoerung.mako

```diff
@@ -0,0 +1,151 @@
+            $('.thumbnail-container').on('click', function (event) {
+              event = event || window.event;
+                event.preventDefault();
+              var target = event.target || event.srcElement,
+                link = target.src ? target.parentNode : target,
+                options = {index: link, event: event, onslide: function(index, slide){
+                    /** a "beautiful" line of code which sets the title of the gallery to the current copyright + photographer of the current photo**/
+                    $('#blueimp-gallery-title').html(($($('.thumbnail-container').children('.thumbnail')[index]).children('div').html()));
+                }},
+                links = this.getElementsByTagName('a');
+              blueimp.Gallery(links, options);
+            });
+        });
+    </script>
+
+
+    <table class="table-with-border kernkraftwerke-extended">
+        <tr>
+            <th class="cell-left">${_('name')}</th>
+            <td>${c['attributes']['zkob'] or '-'}</td>
+        </tr>
+        <tr>
+            <th class="cell-left">${_('kategorie')}</th>
+            <td>${c['attributes']['kategorie'] or '-'}</td>
+        </tr>
+        <tr>
+            <th class="cell-left">${_('tt_kbs_objektart')}</th>
+            <td>
+                % for i, objart in enumerate(objarts):
+                    ${_('kultur' + objart) + ' / ' if (i+1<len(objarts)) else _('kultur' + objart)}
+                % endfor
+            </td>
+        </tr>
+        <tr>
+            <th class="cell-left">${_('tt_kbs_nbr')}</th>
+            <td>${c['featureId'] or '-'}</td>
+        </tr>
+        <tr>
+            <th class="cell-left">${_('grundadresse')}</th>
+            <td>${c['attributes']['adresse'] or ''} ${c['attributes']['hausnr'] or ''}</td>
+        </tr>
+        <tr>
+            <th class="cell-left">${_('tt_kbs_gemeinde')} (${_('tt_kbs_gemeinde_ehemalige')})</th>
+            <td>${c['attributes']['gemeinde'] or ''} ${'('+c['attributes']['gemeinde_ehemalig']+')' if c['attributes']['gemeinde_ehemalig'] else ''}</td>
+        </tr>
+        <tr>
+            <th class="cell-left">${_('Coordinates')}</th>
+            <td>${int(round(c['attributes']['x'],0)) or ''} / ${int(round(c['attributes']['y'],0)) or ''}</td>
+        </tr>
+    % if c['attributes']['pdf_list'] is not None:
+        <tr>
+            <th class="cell-left">${_('Feature tooltip')}:</th>
+            <td>
+	        % for pdf in c['attributes']['pdf_list'].split(','):
+                <a href="https://${dataGeoAdminHost}/${dataPath}/PDF/${pdf}" target="_blank">${pdf}</a><br />
+	        % endfor
+            </td>
+	    </tr>
+    %endif
+    % if c['attributes']['link_uri'] is not None:
+        <tr>
+          <th class="cell-left">${_('legalregulationlink')}</th>
+            <td><a href="${c['attributes']['link_uri']}">${c['attributes']['link_title']}</a></td>
+        </tr>
+    % endif
+    % if c['attributes']['link_2_uri'] is not None:
```

Comment:
looks like these if clause do not work properly the rows `weitere informationen` appeares 3 times out of which two are empty.
https://mf-chsdi3.dev.bgdi.ch/ltbal_kulturgueter/rest/services/ech/MapServer/ch.babs.kulturgueter-anhoerung/3920/extendedHtmlPopup?lang=de

Reply:
This is due to the data delivering an empty string. It is the same for ch.babs.kulturgueter (babs.mako & vbs_master.babs.kgs) which has just been copied. The latter is as well the origin of the other remarks and fancy naming. @mariokeusen is this ok or would it be desirable to adjust this?

Reply:
OK I see, you could check for an empty string as well. But it is probably better to check what is really requested. It just looks a bit strange to me from a user perspective.